testnets/jackaltestnet2/assetlist.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "jackaltestnet2",
+  "assets": [
+    {
+      "description": "The native staking and governance token of Jackal.",
+      "denom_units": [
+        {
+          "denom": "ujkl",
+          "exponent": 0
+        },
+        {
+          "denom": "jkl",
+          "exponent": 6
+        }
+      ],
+      "base": "ujkl",
+      "name": "Jackal",
+      "display": "jkl",
+      "symbol": "JKL",
+      "coingecko_id": "jackal",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/jackal/images/jkl.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/jackal/images/jkl.svg"
+        }
```

Comment:
You can also see how it has image_sync (although I will eventually be generated automatically once the trace is defined)